packages/checkout/src/checkout.ts

```diff
@@ -52,6 +65,7 @@ export default class Checkout extends EventEmitter2 {
     formOnly?: boolean
     walletsOnly?: boolean;
     language?: string;
+    darkMode?: boolean;
```

Comment:
Can we make this into an interface rather tahn defining the types here inline?

Reply:
Yes we can implement an interface, can you create a tech debt ticket for this please?